src/Form/DTO/ChangePasswordDTO.php

```diff
@@ -4,23 +4,29 @@
 
 use Symfony\Component\Security\Core\Validator\Constraints as SecurityAssert;
 use Symfony\Component\Validator\Constraints as Assert;
+use App\Validator\Constraints as CustomAssert;
 
 class ChangePasswordDTO
 {
     /**
      * @var string
      *
-     * @SecurityAssert\UserPassword(message="The current password is incorrect")
+     * @SecurityAssert\UserPassword(message="The current password is incorrect", groups={"changepass"})
```

Comment:
The ChangePasswordDTO class is only for changing passwords, so adding a group for the validation does not make much sense as there are no variations in how the validation is checked.